### ISSUE TITLE
observable on objects / instances

### ISSUE THREAD

**Issue by joakimeriksson** [sics-iot/lwm2m-contiki#13]
Observable on full instances needs to be working to make it easy to subscribe to updates in instances.

**Comment by danielkilbo** [comment]
Pmin, Pmax, Lt, Gt, Step - handling should be handled in the lwm2m-engine (not in the object impl).

**Comment by joakimeriksson** [comment]
Yes, agree. These features should filter any "notifications" from value changes in the objects automatically.
